packages/core/resolve-client/src/client.ts

```diff
@@ -71,11 +71,9 @@ export const command = (
   asyncExec()
     .then((result) => {
       actualCallback(null, result, cmd)
-      return result
     })
     .catch((error) => {
       actualCallback(error, null, cmd)
-      throw error
     })
```

Comment:
If `actualCallback` will throw error, unhandled rejection still may appear. It's better to avoid unhandled rejections anyway

Reply:
It's actualCallback's (user) problem.